src/interface/src/app/treatments/treatment-project-area/treatment-project-area.component.scss

```diff
@@ -7,3 +7,6 @@
     }
 }
 
+::ng-deep .base-maps-selector summary {
```

Comment:
We need to hide the Map selector summary from out MapBaseLayerComponent since it is not required in this design.